hole/__init__.py

```diff
@@ -45,7 +45,7 @@ def __init__(
             async with async_timeout.timeout(5, loop=self._loop):
                 response = await self._session.get(self.base_url)
 
-            _LOGGER.info("Response from *hole: %s", response.status)
+            _LOGGER.debug("Response from *hole: %s", response.status)
```

Comment:
@fabaff , it is possible that the INFO level is by design or required/necessary for some other use case. Please advise.  (if so, was thinking alternative adding a param to the method `quiet` or something.?

Reply:
Looks like that `debug` is sufficient.